pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -700,6 +703,56 @@ var _ = Describe("ImageUpload", func() {
 			validatePVCDefaultInstancetypeLabels()
 		})
 
+		It("Should create DataSource pointing to the PVC", func() {
+			testInit(http.StatusOK)
+			cmd := clientcmd.NewRepeatableVirtctlCommand(
+				commandName, "dv", targetName,
+				"--size", dvSize,
+				"--uploadproxy-url", server.URL,
+				"--insecure",
+				"--force-bind",
+				"--datasource",
+				"--image-path", imagePath,
+			)
+			Expect(cmd()).To(Succeed())
+			Expect(pvcCreateCalled.IsTrue()).To(BeTrue())
+			Expect(kvtesting.FilterActions(&cdiClient.Fake, "create", "datasources")).To(HaveLen(1))
+		})
+
+		It("Should patch DataSource pointing to the PVC", func() {
```

Comment:
This test could assert that the patch was successful, i.e. the expected labels are now present.

Reply:
Done! Thanks Felix :)